notes/returned-mail-received.vbs

```diff
@@ -191,70 +227,64 @@ IF mailing_address_confirmed = "YES" or residential_address_confirmed = "YES" TH
         		err_msg = ""
         		DIALOG Dialog1
         		cancel_confirmation
-        		IF verif_request_checkbox = UNCHECKED and CRF_checkbox = UNCHECKED and SVF_checkbox= UNCHECKED THEN err_msg = err_msg & vbCr & "Please elect     the verification requested and ensure forms are sent in ECF."
-        		IF mets_addr_correspondence = "YES" THEN
+        		IF verif_request_checkbox = UNCHECKED and CRF_checkbox = UNCHECKED and SVF_checkbox= UNCHECKED THEN err_msg = err_msg & vbCr & "Please select the verification requested and ensure forms are sent in ECF."
+				IF mets_addr_correspondence = "Select:" THEN err_msg = err_msg & vbCr & "Please select if correspondence has been sent to METS."
+				IF mets_addr_correspondence = "YES" THEN
         			IF METS_case_number = "" OR (METS_case_number <> "" AND len(METS_case_number) > 10) OR (METS_case_number <> "" AND IsNumeric(METS_case_number) = False) THEN err_msg = err_msg & vbCr & "Please enter a valid METS case number."
         		END IF
-        		IF mets_addr_correspondence = "Select:" THEN err_msg = err_msg & vbCr & "Please select if correspondence has been sent to METS."
         		IF err_msg <> "" THEN MsgBox "*** NOTICE!!! ***" & vbCr & err_msg & vbCr & vbCr & "Please resolve for the script to continue."
         	LOOP UNTIL err_msg = ""
         	CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not pass worded out of MAXIS, allows ser     to password back into MAXIS
         LOOP UNTIL are_we_passworded_out = false					'loops until user passwords back in
 	END IF
 
 	return_mail_checkbox = CHECKED ' it is implied in the title of the script'
-	IF ADDR_actions = "forwarding address outside MN" THEN county_code = "89 Out-of-State"
 	IF ADDR_actions = "forwarding address in MN"  THEN
 		new_addr_state = "MN"
 		reservation_addr = "No"
```

Comment:
Reservation information is not needed here.

Reply:
removed